challenges/classes.js

```diff
@@ -1,7 +1,28 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+class Cuboid {
+  constructor(cubAtt){
+    this.length = cubAtt.length;
+    this.height = cubAtt.height;
+    this.width = cubAtt.width;
+  }
+    volume(length, width, height){
```

Comment:
Here you could use volume() { 
return value here } Consider how this could make your code run more efficiently. Would it? or would it stay the same? Try experimenting.